### ISSUE TITLE
Collaboration Cycle for [Benefits Non-Disability, Burial 530, Form Updates]

### ISSUE THREAD

**Issue by steele-lm** [department-of-veterans-affairs/va.gov-team#65251]
### VFS team name

Benefits Non-Disability

### Product name

Burial Form 530

### Feature name

Burial Form 21P-530 Parity Updates: The current online burial form (v1) is at least two versions behind the paper version of the form, and it's missing fields required for processing. The scope of this project is limited to adding the missing fields as quickly as possible to ensure claims are processable by downstream systems and teams. Efforts will also be focused on hardening the online form to prevent additional silent failures. Future work will focus on: fixing existing accessibility issues, optimizing form flow, and replacing depreciated design system components.

### GitHub label for product

burial, umbrella

### Public DSVA Slack channel for VFS team

#benefits-non-disability

## Kickoff questions

<details><summary>Toggle kickoff questions</summary>

### When did/will you start working on this product/feature?

August 2023

### Will your work result in visible changes to the user experience?

Yes

### Are you doing research with VA.gov users?

No

### Will your work involve changes to...

Both

### Does your product/feature have Google Analytics tracking and a KPI dashboard in Domo?

No

### Do you need to capture any additional analytics or metrics?

Yes

### Product outline

https://github.com/department-of-veterans-affairs/va.gov-team/edit/master/products/pension/530-reactivation-initiative-brief.md

### Verify all relevant materials provided to Governance Team

- [X] I have provided all relevant and up-to-date links, screenshots, images, designs, etc. of the as-is version of the product

### Notify the Collaboration Cycle team of this ticket in your team's public Slack channel.

- [X] I acknowledge that I must notify the Governance team by completing the collab cycle kicckoff workflow in my team's Slack channel after submitting this issue. This can be completed by typing "/collab cycle" and selecting the appropriate workflow.

Kickoff Slack Thread with VFS team: https://dsva.slack.com/archives/C057W3N9K2S/p1694204886692349

</details>

### Design Intent
<details>
  <summary>Toggle Design Intent</summary>

##### VFS actions
- [ ] Review [Design Intent Guidance](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/design-intent) to understand what this touchpoint involves.
- [ ] Schedule your Design Intent (with at least 2 business days lead time from now):
  - Open the [Calendly design intent calendar](https://calendly.com/collaboration-cycle/design-intent-or-midpoint-review)
  - Select a date and time and click "Confirm"
  - Add your name and email
  - Click "Add Guests" and enter the email addresses for VFS team attendees
  - Click "Schedule Event"
- [ ] Link all artifacts in the `Design Intent artifacts for review` section below at least two business days before the scheduled Design Intent. Please don't add artifacts in the comments section.

**Design Intent artifacts**

See Platform guidance on [Design Intent artifacts](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/design-intent#Designintent-Artifacts). Platform feedback is based solely on the artifacts provided, as reviewed during the two days before the Design Intent meeting. Any work not included in the artifacts below or any ongoing work taking place during the review period may not be reflected in that feedback.

- [x] [User flow](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1691512530884/67e3bc6677c8d17cf6fc8848319a6e40ebced1af?wid=0-1694615398728) 
- [ ] [Whiteboard sketch or other lo-fi prototypes or wireframes](https://sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7)
- [ ] Research plan
- [ ] Any other artifacts you have so far
  
##### Platform actions
- [x] [Slack thread with VFS team](https://dsva.slack.com/archives/C057W3N9K2S/p1697122771590129)
- [x] Meeting date/time: 10/19/2023, 1:30ET
  
##### Platform actions

* Accessibility
  * [x] Feedback ticket attached
  * [ ] No feedback
* Design
  * [x] Feedback ticket attached
  * [ ] No feedback
* IA
  * [x] Feedback ticket attached
  * [ ] No feedback
* [x] Update this ticket with the Zoom recording
  - [Design intent recording](https://oddball-io.zoom.us/rec/share/yEXEKkX47GQRceDzFN1t_Egd1B9f6lIuR1-v6_gNvWg6gTbh_oDF13AJwH8E7h_e.EvbzagW6-pe94CNL)
  - Passcode: `@lHby0Yz`

  
##### VFS actions

- [ ] Review feedback tickets and comment on the ticket if there are any questions or concerns
  
</details>

### Sitewide CAIA Intake Request
<details>
  <summary>Toggle Sitewide CAIA Intake Request</summary>

##### VFS actions
- [x] Complete the [Sitewide CAIA intake request process](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Sitewide-CAIA-intake-request.2124906551.html)
- [x] Link to the Sitewide CAIA Intake Request ticket: https://github.com/department-of-veterans-affairs/va.gov-team/issues/68984

</details>

### Midpoint Review
<details>
  <summary>Toggle Midpoint Review</summary>
  
##### VFS actions
Navigate to reference link: [Midpoint Review Guidance](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/midpoint-review)  
- [ ] Schedule your Midpoint Review when ready:
   - Open the [Calendly midpoint review calendar](https://calendly.com/collaboration-cycle/design-intent-or-midpoint-review)
   - Select a date and time and click “Confirm”
   - Add your name and email
   - Click "Add Guests" and enter the VFS meeting attendees email addresses
      - Invite all relevant VFS team members, including accessibility support, product owners, and other VA stakeholders
   - Click "Schedule Event"
- [ ] Check this box if you'd like this review to be asynchronous (Please refer to the [Midpoint Review guidance](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Midpoint-review.1781039167.html) for the difference between a synchronous meeting and an asynchronous review)
- [ ] Link all artifacts **ONLY** in the Midpoint Review artifacts section below at least two days before the scheduled Midpoint Review.  **Do NOT add artifacts to Comments section** 

**Midpoint Review artifacts**

See Platform guidance on [Midpoint Review artifacts](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/midpoint-review#Midpointreview-Artifacts). Platform feedback is based solely on the artifacts provided, as reviewed during the two days before the Midpoint Review meeting. Any work not included in the artifacts below or any ongoing work taking place during the review period may not be reflected in that feedback.

Provide links or documents for the following:

- [x] Finalized design prototype or mockup: [Burial Form Figma File](https://www.figma.com/file/jpJQSCKiGegacdW4RH5uQB/Burial-530-2024-Updates-MVP?type=design&node-id=174-43726&mode=design)
- [x] Specify which pages are included in the review
   - Intro Page
   - Step 1: Claimant Info
   - Step 2: Veteran Info
   - Step 3: Military History
   - Step 4: Benefits Section
   - Step 5: Additional Info and Doc Upload
   - Review Page (NOTE: We have identified some issues with the review page, but it appears to be a Platform-generated component, so it's unclear what improvements we can make. For MVP, we are considering improvements to the review page as out-of-scope.)
- [x] Research plan (does not apply)
- [x] Conversation guide (does not apply)
 
**Content artifacts**
  
- [x] Please include a link to any [Sitewide CAIA](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/sitewide-content-and-ia-intake-request) feedback you received, including a content source of truth, any relevant static content page and entry point updates, and the intake form.
   - [CAIA Intake Request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/68984)

**IA artifacts**

- [x] Please include a link to any [Sitewide CAIA](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/sitewide-caia-intake-request) feedback you received, including an IA review document or intake form.
   - [IA Feedback](https://github.com/department-of-veterans-affairs/va.gov-team/issues/68006)

##### Platform actions
- [x] [Slack thread with VFS team](https://dsva.slack.com/archives/C057W3N9K2S/p1702580457497279)
- [x] Meeting date/time: Tuesday, December 19, 2023, at 11:30 a.m. Eastern

##### Platform actions

* Accessibility
  * [x] Feedback ticket attached
  * [ ] No feedback
* Content
  * [x] Feedback ticket attached
  * [ ] No feedback
* Design
  * [x] Feedback ticket attached
  * [ ] No feedback
* IA
  * [x] Feedback ticket attached
  * [ ] No feedback
* [x] Update this ticket with the Zoom recording
  - Recording link: https://oddball-io.zoom.us/rec/share/NPUcISVmHv1xF5QQgo_s2j5vDe-AsIs2EqEPKk5yamkumBj85lPGyRC_50dIur1c.Absw80NmK9dnbDpz
  - Passcode: PPT%?w71
  
##### VFS actions
  
- [x] Review feedback tickets and comment on the ticket if there are any questions or concerns
  
</details>

### Analytics Request
<details>
  <summary>Toggle Analytics Request</summary>

#### VFS actions
- [ ] Complete the [analytics request process](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Analytics-request.1782120453.html)
- [ ] Link to the Analytics Implementation and QA Request ticket below

</details>

### Contact Center Review
<details>
  <summary>Toggle Contact Center Review</summary>

#### VFS actions
- [x] Complete the [Contact Center review process](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Contact-center-review.1782317061.html)
- [x] Link to the Contact Center review request ticket below: #81826 

</details>

### Staging Review
<details>
  <summary>Toggle Staging Review</summary>
  
##### VFS actions
- Navigate to reference link: [Staging Review Guidance](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/staging-review)  
- [ ] Schedule your Staging Review when ready:
   - Open the [Calendly staging review calendar](https://calendly.com/collaboration-cycle/staging-review)
   - Select a date and time and click “Confirm”
   - Add your name and email
   - Click "Add Guests" and enter the VFS meeting attendees email addresses
      - Invite all relevant VFS team members, including accessibility support, product owners, and other VA stakeholders
   - Click "Schedule Event"
- [ ] If this product contains any [experimental design](https://design.va.gov/experimental-design/), add the `experimental-design` label and schedule a meeting with DSC to present the research findings.
- [ ] Link all artifacts **ONLY** in the Staging Review artifacts section below at least four days before the scheduled Staging Review.  **Do NOT add artifacts to Comments section**
- [ ] I confirm the environment is available and test users have been provided.
- [ ] Please verify your product information in the [Product Directory](https://depo-platform-documentation.scrollhelp.site/getting-started/vfs-product-directory).

**Staging Review artifacts**
  
See Platform guidance on [Staging Review artifacts](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Staging-review.1810137181.html#Stagingreview-Artifacts). Platform findings are based solely on the staging URL and test users provided, as reviewed during the four days before the Staging Review meeting. Any test cases not covered or any ongoing work taking place during the review period may not be reflected in those findings.  

**Product artifacts** 

- [x] Staging URL: https://staging.va.gov/burials-and-memorials/application/530/ (redirects to the new v2 of the form)
- [x] The following pages within the V2 form are included in the review:
   - Intro Page
   - Step 1: Claimant Info
   - Step 2: Veteran Info
   - Step 3: Military History
   - Step 4: Benefits Section
   - Step 5: Additional Info and Doc Upload
   - Review Page
- [x] [Staging test user information](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/burials/530-staging-test-accounts.md)
   - Please note that this form can be completed authenticated or unauthenticated
   - We have also identified some issues within Platform-owned components that will need to be resolved post-launch by the Platform Team (see Notify Platform Team column within the "Review Instance - Unauth" tab in [this spreadsheet](https://docs.google.com/spreadsheets/d/1pKE5rvUgoJe0vkh06W4EVCKL5ygXxgajCRa3nVTLARQ/edit?usp=sharing).)
   
  
**Note:** Please double-check that you've provided staging access information appropriate for testing the tool or feature. Don't put staging credentials in your va.gov-team ticket; store [test user information, passwords, and tasks](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/Administrative/vagov-users/staging-test-accounts-accessible-example.md) in a .md file in the va.gov-team-sensitive repository.
 
**Content artifacts**
- [CAIA Intake Request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/68984)
- [Finalized Burial design prototype in Figma](https://www.figma.com/file/jpJQSCKiGegacdW4RH5uQB/Burial-530-2024-Updates-MVP?type=design&node-id=174-43726&mode=design)

**IA artifacts**
- [x] Please include a link to any [Sitewide CAIA](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/sitewide-caia-intake-request) feedback you received, including an IA review document or intake form.
   - [IA Feedback](https://github.com/department-of-veterans-affairs/va.gov-team/issues/68006)

**QA artifacts**

An artifact that corresponds to each standard in the [QA Standards](https://depo-platform-documentation.scrollhelp.site/developer-docs/quality-assurance-standards).
- [x] [Regression test plan](https://dsvavsp.testrail.io/index.php?/plans/view/5332)
- [x] [Test plan](https://dsvavsp.testrail.io/index.php?/plans/view/5334)
- [x] [Coverage for References](https://dsvavsp.testrail.io/index.php?/reports/view/719)
- [x] [Summary(Defects) reports](https://dsvavsp.testrail.io/index.php?/reports/view/720)
- [x] [E2E tests](https://github.com/department-of-veterans-affairs/vets-website/tree/ndbex-63351-burials-v2/src/applications/burials-v2/tests/e2e) (provide a link to the product’s code)
- [x] [Code coverage](https://github.com/department-of-veterans-affairs/vets-website/tree/ndbex-63351-burials-v2/src/applications/burials-v2/tests) (provide a link to the product’s code)  
  
**Accessibility artifacts**
  
- [x] [Completed accessibility testing artifact](https://github.com/department-of-veterans-affairs/va.gov-team/issues/77395). For details, see [Prepare for an accessibility staging review](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review).
  
##### Platform actions
- [x] [Slack thread with VFS team](https://dsva.slack.com/archives/C057W3N9K2S/p1711656977214559?thread_ts=1711656976.555329&cid=C057W3N9K2S)
- [x] Meeting date/time: ~Friday. March 22 at 2:00 p.m. ET~ ~Rescheduled: Tuesday, March 26, 2024 at 11:30 a.m. Eastern~ ~Rescheduled: Friday, March 29, 2024 at 2.00PM ET~ ~Rescheduled: Thursday, April 4 at 1.30 PM ET~ Rescheduled: Friday, April 5 at 2 p.m. Eastern

##### Platform actions

- [x] Update this ticket with the Zoom recording
  - [Recording link](https://oddball-io.zoom.us/rec/share/l0UrGixPBGbIq5cJxO-JY5NE5HD62tpdINjGNJVvVxvSxAQ2Nsxx-CbkzQaj3TpN.pdcWzAxdXM3yTIQA)
  - Password: `aSHZi6#n`
  
##### VFS actions
  
- [x] Review the findings tickets and comment on the ticket if there are any questions or concerns
- [x] Close individual findings tickets when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment on the issue ticket.
- [x] After launch, [request an accessibility audit from the VA 508 Office](https://depo-platform-documentation.scrollhelp.site/developer-docs/request-support-from-the-va-508-office#RequestsupportfromtheVA508office-AuditRequest). This is required even if no accessibility issues were found during the Staging Review.
  - [x] Share ServiceNow ticket number here: REQ27929555
- [ ] Close ticket once Privacy, Security, Infrastructure Readiness Review has been completed, VA 508 Office audit is requested, and all other post-Staging actions are complete
</details>

### Privacy, Security, Infrastructure Readiness Review
<details>
  <summary>Toggle Privacy, Security, Infrastructure Readiness Review</summary>
  
#### VFS actions
- [x] Complete the [Privacy, security, infrastructure readiness review process](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Privacy-and-security-review.1782317101.html)
- [x] [Readiness Review ticket](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/issues/1805)

#### Platform actions
- [x] Privacy, security, infrastructure readiness review is complete

</details>

**Comment by fiorella-io** [comment]
Since we are still working with CAIA to finalize the content in the form, is it safe to start the mid-point review if the content on our designs is not final?

**Comment by steele-lm** [comment]
@shiragoodman I wanted to check in to see if this ticket needed to (or could be) closed. We have two outstanding issues:
- #80114 is blocked by an update the Design System Team is working on. Once that update is made, the breadcrumb component can be implemented.
- #80105 design work has been completed and approved, and a new team will be implementing. (Maintenance of the burial form is moving to the team that manages the pension 527EZ form). cc @matt4su

**Comment by shiragoodman** [comment]
@steele-lm yes, you're free to close the Collab Cycle Request ticket even if there's still open issues. Closing this ticket doesn't automatically close the issue tickets, so there's no conflict. Thank you for checking.
